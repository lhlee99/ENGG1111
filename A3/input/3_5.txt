3
Eric -2.8 3 3.2
Kyle 1 0 1
Kenny -1 -3.4 5.2
