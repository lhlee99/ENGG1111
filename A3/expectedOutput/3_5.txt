Eric: Kenny
Kyle:
Kenny: Eric
